src/wallet/wallet.cpp

```diff
@@ -2173,6 +2172,7 @@ bool CWallet::CommitTransaction(CWalletTx& wtxNew, CReserveKey& reservekey)
             if (!wtxNew.AcceptToMemoryPool(false))
             {
                 // This must not fail. The transaction has already been signed and recorded.
+                // Also the fee never exceeds maxTxFee which forces mempool rejection
```

Comment:
Guess I will drop this because it is already mentioned in `wallet.h`: https://github.com/bitcoin/bitcoin/pull/7084/files#diff-12635a58447c65585f51d32b7e04075bR205